firebase-config/src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -773,6 +791,41 @@ void updateAbtWithActivatedExperiments(@NonNull JSONArray abtExperiments) {
     return experimentInfoMaps;
   }
 
+  /**
+   * Notifies the Firebase Personalization SDK about a get() call.
+   *
+   * @hide
+   */
+  void pullArm(String key) {
+    if (firebasePersonalization != null) {
+      // If there is no firebasePersonalization instance, then this FRC is either in a non-3P
+      // namespace or in a non-main FirebaseApp, so there is no reason to call Personalization.
+      // For more info: RemoteConfigComponent#isAbtSupported.
+      executor.execute(() -> firebasePersonalization.pullArm(key));
+    }
+  }
+
+  /**
+   * Notifies the Firebase Personalization SDK about activated metadata.
+   *
+   * @hide
+   */
+  void updatePersonalizationWithActivatedMetadata(@NonNull ConfigContainer configs) {
```

Comment:
It is not clear why the personalization SDK needs to know anything at this stage. That seems like an optimization

Reply:
Yeah, I guess it is. Removed.